src/modules/auth/views/auth-view.vue

```diff
@@ -1,3 +1,25 @@
 <template>
-  <div>AuthView</div>
+  <div class="auth--wrapper">
+    <div class="auth--wrapper__form--container">
```

Comment:
Klasy do poprawy jak wyżej, to jakiś mix BEM trochę średnio konsekwentny.
Może być po prost auth-wrapper a ponizej auth-wrapper__form

Reply:
albo po prostu auth a w nim auth__form